plugins/flipping-utilities

```diff
@@ -1,2 +1,3 @@
-repository=https://github.com/Belieal/flipping-utilities.git
-commit=09b17807a785285512be2d87ef286c74ead963ae
\ No newline at end of file
+repository=https://github.com/Flipping-Utilities/rl-plugin.git
+commit=40583d4a673a8d114b082769e922827d1590b638
+warning=If you've explicitly opted in to our syncing feature by logging in to our api, your GE data will be synced with our api. If you do not opt in, all of your data remains stored locally and the plugin still remains functional.
```

Comment:
Keep in mind that this message should be written from the perspective of RuneLite. With that in mind, this is not an acceptable warning message format--you should look to other examples in the hub, such as `This plugin submits Grand Exchange transactions and your IP address to a 3rd party website not controlled or verified by the RuneLite Developers`

Reply:
Is this new text fine?
If not, we're open to suggestions: Trying to show that it is optional, while keeping the warning information relevant if they do log in.

> If you opt in to do so, this plugin will submit Grand Exchange transactions and your IP address to a 3rd party website not controlled or verified by the RuneLite Developers. If you do not opt in, this plugin will store your data only locally and will still remain functional.